ImageGallery/src/org/sleuthkit/autopsy/imagegallery/datamodel/DrawableDB.java

```diff
@@ -688,6 +704,65 @@ private void insertOrUpdateFile(DrawableFile f, @Nonnull DrawableTransaction tr,
         }
     }
 
+
+   /**
+     * Gets all known data source object ids from data_sources table
+     *
+     * @return list of known data source object ids
+     */
+    public Set<Long> getKnownDataSourceIds() throws TskCoreException {
```

Comment:
I think this should just be getDataSourceIds. "Known" is too overloaded a term to use here.